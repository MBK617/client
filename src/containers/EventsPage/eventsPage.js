import React from 'react';
import { View, Text } from "react-native";

const EventsPage = () => {
  return (
    <View>
      <Text>Events</Text>
    </View>
  );
}

export default EventsPage;