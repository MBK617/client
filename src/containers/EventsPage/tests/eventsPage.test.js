import React from 'react';
import { render, cleanup } from '@testing-library/react-native';

import EventsPage from '..';

describe('<EventsPage />', () => {
  let rendered;
  beforeEach(() => {
    rendered = render(<EventsPage />);
  })

  afterEach(cleanup);

  it('should render correctly', () => {
    const tree = rendered.toJSON();
    expect(tree).toMatchSnapshot();
  });
});