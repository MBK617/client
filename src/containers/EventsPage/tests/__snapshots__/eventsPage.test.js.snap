// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<EventsPage /> should render correctly 1`] = `
<View>
  <Text>
    Events
  </Text>
</View>
`;
